### DISCUSSION TITLE
Do you want all Infinix inbook series support (telegram Group) ?

### DISCUSSION THREAD

**Discussion by devboloji** [devboloji/Infinix-Hackintosh-Opencore-Guide#15] [Polls]
Many are facing the issues and asking me doubts about Infinix Hackintosh. So support group will be good idea for this. If you want support and help to all Infinix inbook series support (telegram Group) you can answer this poll. Telegram group is better than Github issues tab. Because there will be delay in raising the issues to notify me in github. So, this is the reason.

**Comment by ghost** [comment]
Is there any activity going on inside that group? I have never used Telegram before, but it would be interesting if I did!

**Reply by devboloji** [reply]
@bunnybavros actually most of the people doesn't want to join Telegram Group. So, the telegram group is shut down. Telegram group is same like whatsapp group. In telegram group, you get fast response for any issue.

**Reply by ghost** [reply]
Sticking with GitHub could be a solid choice. 
To be honest, I fell in love with GitHub because of your work and I hope to support you in some way in the future.

**Comment by devboloji** [comment]
@bunnybavros  Ya brother. Thanks for supporting and helping me with quick helping others issues. Thank you..
